src/pages/lib/auth/mfa/q/platform/[platform].mdx

```diff
@@ -1,15 +1,778 @@
 export const meta = {
-  title: `Multi-factor authentication`,
+  title: `Manage MFA settings`,
   description: `Learn how to enable multi-factor authentication with Amplify.`,
 };
 
```

Comment:
@abdallahshaban557 Just noting here that this updated guide does not include the current advanced use cases content here - https://docs.amplify.aws/lib/auth/mfa/q/platform/js/#advanced-use-cases Will want to review if we want to readd this in expanders or move to another page

Reply:
yes, that is fine. We will want to have that be documentation we add seperately. Possibly to the custom auth page we create. It is too confusing to add here.

Reply:
can we have this in our backlog, so that we can review it later?

Reply:
Yes these are noted and we can revisit later